scripts/workers/submission_worker.py

```diff
@@ -277,12 +283,61 @@ def run_submission(challenge_id, challenge_phase, submission_id, submission, use
     # call `main` from globals and set `status` to running and hence `started_at`
     submission.status = Submission.RUNNING
     submission.save()
-    with stdout_redirect(stdout) as new_stdout, stderr_redirect(stderr) as new_stderr:      # noqa
-        submission_output = EVALUATION_SCRIPTS[challenge_id].evaluate(annotation_file_path,
-                                                                      user_annotation_file_path,
-                                                                      challenge_phase.codename,)
+    try:
+        successful_submission_flag = True
+        with stdout_redirect(stdout) as new_stdout, stderr_redirect(stderr) as new_stderr:      # noqa
+            submission_output = EVALUATION_SCRIPTS[challenge_id].evaluate(annotation_file_path,
+                                                                          user_annotation_file_path,
+                                                                          challenge_phase.codename,)
+
```

Comment:
write a proper comment explaining the criteria of marking a submission as successful. This will help any one digging or updating code in future.